roles/kubernetes/node/templates/azure-cloud-config.j2

```diff
@@ -9,5 +9,14 @@
   "securityGroupName": "{{ azure_security_group_name }}",
   "vnetName": "{{ azure_vnet_name }}",
   "vnetResourceGroup": "{{ azure_vnet_resource_group }}",
-  "routeTableName": "{{ azure_route_table_name }}"
+  "routeTableName": "{{ azure_route_table_name }}",
+{% if azure_primary_availability_set is defined %}
+  "primaryAvailabilitySetName": "{{ azure_primary_availability_set }}",
+{%endif%}
+  "useInstanceMetadata": {{azure_use_instance_metadata }},
+{% if azure_loadbalancer_sku is standard %}
```

Comment:
Shouldn't this be `azure_loadbalancer_sku  == 'standard'` ?

Reply:
my bad i havent pushed the last version of the code :/. Should be good now

Reply:
@Miouge1  can you please take a look again. Thank you